### ISSUE TITLE
Allow use of podman instead of docker

### ISSUE THREAD

**Issue by ssbarnea** [wemake-services/docker-image-size-limit#223]
This python package is awesome but currently it cannot be used with podman but it should be quite easy to make it recognize podman, or at least to have an option to make use of podman instead of docker as their APIs are compatible.

**Comment by sobolevn** [comment]
Good idea! Probably we can remove `docker` package and use `subprocess` with both `docker` and `podman` commands.

**Comment by ssbarnea** [comment]
I did not look at the code yet but it might be a good idea. For example I had to add paramiko to dependecies because my docker was remote and in that case is needed.

It might take some time but now that I known that podman support is welcomed, I might  attempt to make a PR.

**Comment by sobolevn** [comment]
This would be awesome! Thank you!
